pwa/components/repairers/RepairersResults.tsx

```diff
@@ -97,6 +134,8 @@ export const RepairersResults = (): JSX.Element => {
           <MapContainer
             center={mapCenter}
             zoom={12}
+            // @ts-ignore
+            ref={setMap}
             scrollWheelZoom={false}
             style={{
```

Comment:
```suggestion
                        style={{
              zIndex: 1,
              height: '100%',
              borderRadius: 5,
            }}>
            <UpdateViewOnCitySearch />
            <UpdateZoomOnRadiusSearch />
            <TileLayer url="https://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png" />
```
Like this way and put the behavior in the components

Reply:
ex:
```
 const UpdateViewOnCitySearch = () => {
    const map = useMap();

    if (city) {
      map.setView([Number(city.lat), Number(city.lon)]);
    }

    return null;
  };

  const UpdateZoomOnRadiusSearch = () => {
    const map = useMap();

    switch (searchRadius) {
      case '5000':
        map.setZoom(12);
        break;
      case '10000':
      case '15000':
      case '20000':
      case '30000':
        map.setZoom(11);
        break;
      case '40000':
        map.setZoom(10);
        break;
    }

    return null;
  };
  ```